doc/release/upcoming_changes/28925.deprecation.rst

```diff
@@ -0,0 +1,11 @@
+Setting the ``strides`` attribute is deprecated
+-----------------------------------------------
+Setting the strides attribute is now deprecated since mutating
+an array is unsafe if an array is shared, especially by multiple
+threads.  As an alternative, you can create a new view (no copy) via:
+* `np.lib.stride_tricks.strided_window_view` if applicable,
+* `np.lib.stride_tricks.as_strided` for the general case,
+* or the `np.ndarray` constructor (``buffer`` is the original array) for a light-weight version.
```

Comment:
I do not understand this one. Can it be clarified?

Reply:
`np.ndarray` is an oddly low-level function, but I suspect you can replace most (all?) calls with it:
```
In [8]: arr = np.arange(10)

In [9]: np.ndarray(arr.shape, arr.dtype, arr, strides=8)
Out[9]: array([0, 1, 2, 3, 4, 5, 6, 7, 8, 9])
```
(Change `strides`/`shape` to something else...)